mm2src/mm2_metrics/src/mm_metrics.rs

```diff
@@ -419,21 +410,21 @@ mod test {
         mm_gauge!(metrics, "rpc.connection.count", 3.0, "coin" => "KMD");
         mm_gauge!(metrics, "rpc.connection.count", 5.0, "coin" => "KMD");
 
-        // mm_timing!(metrics,
-        //            "rpc.query.spent_time",
-        //            // ~ 1 second
-        //            34381019796149, // start
-        //            34382022725155, // end
-        //            "coin" => "KMD",
-        //            "method" => "blockchain.transaction.get");
-        //
-        // mm_timing!(metrics,
-        //            "rpc.query.spent_time",
-        //            // ~ 2 second
-        //            34382022774105, // start
-        //            34384023173373, // end
-        //            "coin" => "KMD",
-        //            "method" => "blockchain.transaction.get");
+        let delta = 34382022725155.0 - 34381019796149.0;
```

Comment:
Could you please replace these huge numbers with just `delta = 1.0`?
Or maybe it's wrong, and then you could use `delta = Duration::from_secs(1)`.

Previously, `mm_timing` took a timestamp, now it takes types that implement `IntoF64`, so `Duration` is acceptable as well as `f64`

Reply:
`delta = Duration::from_secs(1)` looks good